monitors.go

```diff
@@ -69,7 +68,7 @@ func (m *Monitor) JobUpdate(updateKey aurora.JobUpdateKey, interval int, timeout
 	}
 
 	fmt.Println("Timed out")
```

Comment:
Good point! Let's leave it as is so that we are able to tell the difference between timing out and an actual error.